netlify.toml

```diff
@@ -144,3 +144,9 @@ HUGO_ENABLEGITINFO = "true"
   status = 301
   force = false
 
+[[redirects]]
+  from = "/docs/reference/artifacts/*"
+  to = "/docs/reference/artifacts-legacy/:splat"
+  status = 301
+  force = false
+
```

Comment:
```suggestion
[[redirects]]
  from = "/reference/artifacts-with-artifactsrewrite/*"
  to = "/docs/reference/ref-artifacts/:splat"
  status = 301
  force = false
```

Reply:
@aimeeu I'm not sure if I'm doing this right but I tried suggesting the addition here. I don't think I can approve the change since I was the one who opened the PR.

Reply:
```suggestion
[[redirects]] 
  from = "/reference/artifacts-with-artifactsrewrite/ *" 
  to = "/docs/reference/ref-artifacts/:splat"
  status = 301 
  force = false
```

Reply:
@nikema-armory can you accept the suggested change I suggested?

Reply:
I don't see how to. I just made a new commit with the changes